include/deal.II/base/mpi.h

```diff
@@ -1463,6 +1482,45 @@ namespace Utilities
 #  endif
     }
 
+    template <typename T>
+    T
+    bcast(const MPI_Comm &   comm,
+          const T &          object_to_send,
+          const unsigned int root_process)
+    {
+#  ifndef DEAL_II_WITH_MPI
+      (void)comm;
+      (void)root_process;
+      return object_to_send;
+#  else
+      const auto n_procs = dealii::Utilities::MPI::n_mpi_processes(comm);
+      const auto my_rank = dealii::Utilities::MPI::this_mpi_process(comm);
+
+      AssertIndexRange(root_process, n_procs);
+
+      std::vector<char> buffer      = Utilities::pack(object_to_send, true);
```

Comment:
```suggestion
      std::vector<char> buffer      = Utilities::pack(object_to_send, false);
```
(often, not compressing is faster)

Reply:
We could also leave the choice to the user if they want compression with another parameter that defaults to false, but I think this would be overkill in this context.